doc/integrator/upgrade_application.rst.tmpl

```diff
@@ -63,6 +63,9 @@ Then follow the instructions.
 From a version 2.5 and next
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Have a look about the important information that you can found in the
```

Comment:
Have a look at the information in the

Reply:
=>
Have a look at the ``Information to know before starting the upgrade`` chapter in the